### PR TITLE
feat: explore teaser

### LINKED ISSUE #16: Explore Teaser
Implement explore teaser block:
- columns / rows
- first cell rendered with background image and link / text on top
- images except first cell are linked to same as titles
- arbitrary link bottom right
- back to top link

Content so far:
Screenshot 2023-02-09 at 08 18 51

How it looks on https://www.choicehotels.com/explore

Mobile:
Screenshot 2023-02-09 at 08 20 02
Screenshot 2023-02-09 at 08 20 12

Desktop:
Screenshot 2023-02-09 at 08 20 27

### PR DESCRIPTION
**Pull request by dzehnder**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #16 

Test URLs:
- Before: https://main--choice--hlxsites.hlx.page/en-us/explore/
- After: https://16-explore-teaser--choice--hlxsites.hlx.page/en-us/explore/

### CODE REVIEW

**Comment by solaris007** [comment]
Visual differences:

Screenshot 2023-02-13 at 20 46 43
Screenshot 2023-02-13 at 20 47 43

Back to top missing:
Screenshot 2023-02-13 at 20 47 10

Clipping and oversized images:
Screenshot 2023-02-13 at 20 48 25
Screenshot 2023-02-13 at 20 48 51

**Comment by solaris007** [comment]
Back to top visual differences (left original)
Screenshot 2023-02-14 at 14 22 19

**Comment by dzehnder** [comment]
@solaris007 refresh the page, just pushed that one :)

**Comment by solaris007** [comment]
Padding title / text still off:
Screenshot 2023-02-14 at 14 23 20

**Comment by solaris007** [comment]
Main image height seems different. should be positioned the same.
Screenshot 2023-02-14 at 14 25 26

**Comment by solaris007** [comment]
same here:
Screenshot 2023-02-14 at 14 27 15

File: blocks/explore-teaser/explore-teaser.css [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+}
+
+.explore-teaser {
+    padding: 30px 30px 30px 0;
+}
+
+.explore-teaser > div:nth-child(2) h2 {
+    font-size: var(--body-font-size-xl);
+    margin: 0;
+    padding: 0 20px 20px;
```
Comment (solaris007) [inline blocks/explore-teaser/explore-teaser.css:12]: --spacing-m

File: blocks/explore-teaser/explore-teaser.css [hunk 2]
```diff
@@ -0,0 +1,217 @@
+.explore-teaser-wrapper {
+    padding: 0 15px
+}
+
+.explore-teaser {
+    padding: 30px 30px 30px 0;
```
Comment (solaris007) [inline blocks/explore-teaser/explore-teaser.css:6]: --spacing-l

File: blocks/explore-teaser/explore-teaser.css [hunk 3]
```diff
@@ -0,0 +1,217 @@
+.explore-teaser-wrapper {
+    padding: 0 15px
```
Comment (solaris007) [inline blocks/explore-teaser/explore-teaser.css:2]: --spacing-s

File: blocks/explore-teaser/explore-teaser.css [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+    color: var(--link-color);
+    text-align: left;
+    white-space: break-spaces;
+    margin: 0;
+    padding: 0;
+}
+
+.explore-teaser > div:nth-child(2) p {
+    margin: 0;
+    padding: 0 20px 20px;
```
Comment (solaris007) [inline blocks/explore-teaser/explore-teaser.css:34]: --spacing-m

File: blocks/explore-teaser/explore-teaser.css [hunk 5]
```diff
@@ -0,0 +1,217 @@
...
+    display: flex;
+    align-items: flex-start;
+}
+
+.explore-teaser > div:first-child > div:first-child {
+    display: flex;
+    justify-content: flex-end;
+    background-size: cover;
+    background-position: 50% 50%;
+    margin-bottom: 30px;
```
Comment (solaris007) [inline blocks/explore-teaser/explore-teaser.css:57]: --spacing-l

**Review by solaris007 — DISMISSED** [review]

**Comment by solaris007** [comment]
congrats, that was a beast to tackle!

**Review by solaris007 — APPROVED** [review]

**Pull request merged**